recipes/riboseed/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "riboSeed" %}
+{% set version = "0.4.16" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "abb680a514058c2233f34a3c574727f61ed65e06b2123964b359683ccf9eb9f5" %}
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  # url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://test-files.pythonhosted.org/packages/21/59/ef9dbb631f132c312bdd1cc145395050fdd0bfbe46765280c9a157367dcd/riboSeed-0.4.16.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  skip: True  # [win]
+  skip: True  # [py2k]
+  skip: True  # [py34]
+  script: $PYTHON setup.py install --single-version-externally-managed --record=record.txt
+requirements:
+  build:
+    - python
+    - setuptools
+    - biopython ==1.68
```

Comment:
Why are you pinning all packages this strictly, is there a reason? Otherwise, just write `biopython` and `pysam` etc...

Reply:
There is a rather difficult web of dependency availability and congruency.  Having a more updated version of pysam doesn't work with the versions of numpy that work well, and so on.  Its a product of the available versions in conda-forge and bioconda.  I hope to get things sorted in the next minor release that will make the packages less strictly pinned.